<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Olivier Trichet &lt;nive@nivalis.org&gt;</author>
 <class>KNode::IdentityWidget</class>
 <widget class="KCModule" name="KNode::IdentityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>482</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="mUseSpecificLayout">
     <item>
      <widget class="QCheckBox" name="mUseSpecificIdentity">
       <property name="text">
        <string>Use a specific identity</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpace1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mSelectorLayout">
     <item>
      <widget class="QLabel" name="mIdentitySelectorLabel">
       <property name="text">
        <string comment="@label:listbox">Identity:</string>
       </property>
       <property name="buddy">
        <cstring>mIdentitySelector</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KNode::IdentityComboWrapper" name="mIdentitySelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="mModifyIdentitiesButton">
       <property name="text">
        <string>Modify...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpace2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="mPreviewLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="mNameLabel">
       <property name="text">
        <string comment="@label:textbox Name of the user in a message">Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mOrganisationLabel">
       <property name="text">
        <string>Organization:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mEmailLabel">
       <property name="text">
        <string>Email address:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mReplytoLabel">
       <property name="text">
        <string>Reply-to address:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mMailcopiestoLabel">
       <property name="text">
        <string>Mail-copies-to:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="mName">
       <property name="text">
        <string notr="true">xxx</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="mOrganisation">
       <property name="text">
        <string notr="true">xxx</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="mMailcopiesto">
       <property name="text">
        <string notr="true">xxx</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="mReplyto">
       <property name="text">
        <string notr="true">xxx</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="mEmail">
       <property name="text">
        <string notr="true">xxx</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpace3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KCModule</class>
   <extends>QWidget</extends>
   <header location="global">kcmodule.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KNode::IdentityComboWrapper</class>
   <extends>QComboBox</extends>
   <header>configuration/identity_combo_wrapper.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mUseSpecificIdentity</tabstop>
  <tabstop>mIdentitySelector</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
